core-developers/memorialization.rst

```diff
@@ -41,8 +41,9 @@ The PSF staff does not follow up with GitHub with regards to GitHub account
 cancellation as this action is reserved for next-of-kin or designated by
 the deceased GitHub user to act as an account successor.
 
-The general policy regarding deceased users on GitHub is described
-`here <https://docs.github.com/en/site-policy/other-site-policies/github-deceased-user-policy>`_.
+The general policy regarding deceased users on GitHub is described on their
+`GitHub Deceased User Policy <https://docs.github.com/en/site-policy/other-site-policies/github-deceased-user-policy>`_
```

Comment:
Based on what we discussed, it would be better to include "GitHub" in the link, but the current sentence repeats it twice.  I would either remove the other one:
```suggestion
The general policy regarding deceased users is described on their
`GitHub Deceased User Policy <https://docs.github.com/en/site-policy/other-site-policies/github-deceased-user-policy>`_ page.
```
or simply:
```suggestion
See also the `GitHub Deceased User Policy
<https://docs.github.com/en/site-policy/other-site-policies/github-deceased-user-policy>`_ page.
```